src/file-picker/src/FilePicker.js

```diff
@@ -157,4 +164,17 @@ export default class FilePicker extends PureComponent {
   handleButtonClick = () => {
     this.fileInput.click()
   }
+
+  handleBlur = e => {
+    const { onBlur } = this.props
+    const { files } = this.state
+
+    // Redux-form derives field value from bubbled onBlur event, which is fake here
+    if (e && e.stopPropagation) e.stopPropagation()
+    if (this.fileInput) this.fileInput.blur()
```

Comment:
Checks are needed only for tests, otherwise we'd have to introduce jsdom in tests.

Reply:
Prefer new lines and curly braces for consistency.